ui/src/app/pages/workspace/invalid-billing-banner-maybe.tsx

```diff
@@ -240,6 +240,14 @@ export const InvalidBillingBannerMaybe = fp.flow(
     isEligibleForExtension
   );
 
+  const showBanner =
+    !showExtensionModal &&
+    creatorUser?.givenName &&
+    creatorUser?.familyName &&
+    ((!workspace.initialCredits.expirationBypassed &&
```

Comment:
I think it would be easier to follow this logic if `isExhausted` were listed first.

Reply:
similar to workspace-utils L25